deploy/charts/harvester/values.yaml

```diff
@@ -65,7 +65,7 @@ kubevirt:
 
       ## Specify kubevirt feature gates
       developerConfiguration:
-        featureGates: ["LiveMigration", "HotplugVolumes"]
+        featureGates: ["LiveMigration", "HotplugVolumes", "HostDevices"]
```

Comment:
do we also need to enable the GPU feature gate? as I remember it was one of the requested feature support.

Reply:
No, we can support GPUs using only PCI Passthrough. vGPU is a separate feature